includes/class-wc-payments.php

```diff
@@ -766,6 +766,7 @@ public static function install_actions() {
 		if ( version_compare( WCPAY_VERSION_NUMBER, get_option( 'woocommerce_woocommerce_payments_version' ), '>' ) ) {
 			do_action( 'woocommerce_woocommerce_payments_updated' );
 			self::update_plugin_version();
+			self::set_plugin_activation_timestamp();
```

Comment:
Not a big deal and this is 100% correct but just to keep this method focused on firing the hook and updating the stored version and not anything else, maybe we could actually hook the call to `set_plugin_activation_timestamp()` in `WC_Payments::init()` like so?

```php
add_action( 'woocommerce_woocommerce_payments_updated', [ __CLASS__, 'set_plugin_activation_timestamp' ] );
```

Reply:
That's a good suggestion 👍 I'll make an update.

Reply:
@waclawjacek Updated in https://github.com/Automattic/woocommerce-payments/pull/2313/commits/1de9974dbd188633bf19ab84531459307c21aaf5